api/test/e2enew/service/service_test.go

```diff
@@ -462,3 +463,83 @@ var _ = ginkgo.Describe("service update use patch method", func() {
 		})
 	})
 })
+
+var _ = ginkgo.Describe("test service delete", func() {
+	t := ginkgo.GinkgoT()
+	var createServiceBody map[string]interface{} = map[string]interface{}{
+		"name": "testservice",
+		"upstream": map[string]interface{}{
+			"type": "roundrobin",
+			"nodes": []map[string]interface{}{
+				{
+					"host":   base.UpstreamIp,
+					"port":   1980,
+					"weight": 1,
+				},
+			},
+		},
+	}
+	_createServiceBody, err := json.Marshal(createServiceBody)
+	assert.Nil(t, err)
+
+	table.DescribeTable("test service delete",
```

Comment:
is there any test for DP？

Reply:
Yes.The DP test are here, you can see:

https://github.com/apache/apisix-dashboard/blob/01c952803d256c53ac411ca52d1c90a2263879fc/api/test/e2e/route_service_upstream_test.go#L93

Reply:
but not in this file

Reply:
And here, there are in this file: 

https://github.com/apache/apisix-dashboard/blob/01c952803d256c53ac411ca52d1c90a2263879fc/api/test/e2enew/service/service_test.go#L93

https://github.com/apache/apisix-dashboard/blob/01c952803d256c53ac411ca52d1c90a2263879fc/api/test/e2enew/service/service_test.go#L193

https://github.com/apache/apisix-dashboard/blob/01c952803d256c53ac411ca52d1c90a2263879fc/api/test/e2enew/service/service_test.go#L310

Reply:
not test DP for this new test case

Reply:
IMO, different test cases test different features.
The former case tests the feature of creating a service and test it in DP.
This test case is testing for the relation of service and upstream (we can't delete service when the service is used by routes.

If this case tests the DP, I think the former test doesn't have mean to exist.

Reply:
Add test DP for this case.